migrations/v2.js

```diff
@@ -0,0 +1,225 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-assessment - v2.0.0 > v2.0.3', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.0.0', { name: 'adapt-contrib-assessment', version: '<2.0.3' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    return assessments.length;
+  });
+
+  /**
+   * * Add field to each item in a JSON array and set blank.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._questions._canShowModelAnswer attribute', async () => {
+    assessments.forEach(assessment => {
+      assessment._questions.forEach(item => {
+        item._canShowModelAnswer = true;
+      });
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._questions._canShowModelAnswer attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._questions.every(item =>
+        item._canShowModelAnswer !== false
+      )
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowModelAnswer not added to every instance of assessment._questions');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-assessment - update to v2.0.3', { name: 'adapt-contrib-assessment', version: '2.0.3', framework: '>=2.0.0' });
+});
+
+describe('adapt-contrib-assessment - v2.0.3 > v2.1.0', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.0.3', { name: 'adapt-contrib-assessment', version: '<2.1.0' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    return assessments.length;
+  });
+
+  /**
+   * * Add JSON field and set attribute.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._suppressMarking', async () => {
+    assessments.forEach(assessment => {
+      assessment._suppressMarking = true;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._suppressMarking attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._suppressMarking === true
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _suppressMarking not added to every instance of assessment as true.');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-assessment - update to v2.1.0', { name: 'adapt-contrib-assessment', version: '2.1.0', framework: '>=2.0.18' });
```

Comment:
The update to `v2.1.0` requires framework `v2.1.3`.

Reply:
Good looking out. Thanks.